tests/testdata/helper-austraits.R

```diff
@@ -164,11 +175,23 @@ test_list_names_valid <- function(data, info) {
 }
 
 
-test_list_named <- function(data, expected_names, info) {
+test_list_named_exact <- function(data, expected_names, info) {
   test_list_names_valid(data, info)
   expect_named(data, expected_names, info= info)
 }
 
+test_list_named_allowed <- function(data, expected_names, info) {
+  test_list_names_valid(data, info)
+  expect_named(data)
+  expect_contains(expected_names, names(data), info= info)
+}
+
+test_list_named_contains <- function(data, expected_names, info) {
+  test_list_names_valid(data, info)
+  expect_named(data)
+  expect_contains(names(data), expected_names, info= info)
+}
+
 test_list_named_contains <- function(data, expected_names, info) {
   test_list_names_valid(data, info)
```

Comment:
Why are there two functions names `test_list_names_contains`. Was the newly added one meant to overwrite the old one?

Reply:
Whoops, Thanks